src/api-service/__app__/onefuzzlib/tasks/main.py

```diff
@@ -125,7 +124,7 @@ def init(self) -> None:
 
     def stopping(self) -> None:
         logging.info("stopping task: %s:%s", self.job_id, self.task_id)
-        ProxyForward.remove_forward(self.task_id)
+        delete_queue(str(self.task_id), StorageType.corpus)
```

Comment:
Why are you calling `delete_queue` here?

Reply:
This line was already present. I did not make any changes.

Reply:
Your PR re-adds this, as it was removed in a previous commit.

Reply:
I see. I'll make sure to remove.